index.js

```diff
@@ -121,27 +121,7 @@ function convertDependencies(schema) {
   }
 
   for (const key in deps) {
-    const foo = {
-      oneOf: [
-        {
-          not: {
-            required: [key],
-          },
-        },
-        {
-          required: [].concat(key, deps[key]),
-        },
-      ],
-    };
-
-    if (typeof deps[key] === "object") {
```

Comment:
I actually have this change locally but not committed, though it keeps the above above `const foo =` section, and only drops this if block. @HatefulMoron I might try committing the latest that I was using if you can test if that works as expected? or do you think the rest needs to go as well? Then needs to be pushed to npm I guess